examples/asynchronous_sleep.cpp

```diff
@@ -1,4 +1,18 @@
+/*
+Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
The comment blocks in this diff don't match the existing ones (e.g. [in `then.hpp`](https://github.com/facebookexperimental/libunifex/blob/main/include/unifex/then.hpp).